R/Model-class.R

```diff
@@ -1,4 +1,4 @@
-#' @include helpers.R Model-validity.R
+#' @include logger.R helpers.R helpers_jags.R Model-validity.R
```

Comment:
better have new line for each included file (easier to read)

Reply:
Actually, these 2 are not needed as we export all of the functions.